backend/src/app.js

```diff
@@ -299,7 +305,16 @@ router.post('/v1/templates/:name/send/:stmpSelected?', async (ctx) => {
         html: getHtml(mjmlTemplatePath, ctx.request.body),
         text: getTxt(txtTemplatePath, ctx.request.body),
         attachments: attachments
-    });
+    }, isNil));
+
+    const smtp1 = config.get('smtp');
+    const smtp2 = config.get('smtp2');
+    if (ctx.params.stmpSelected === 'smtp1' && smtp1.awsSesMessageIdHost) {
+        result.messageId = `${result.response.split(' ')[2]}@${smtp1.awsSesMessageIdHost}`;
```

Comment:
@Its-Alex **suggestion:** 

```suggestion
        result.messageId = `${result.response.split(' ')[2]}@${smtp1.awsSesMessageIdHost}` ?? null;
```

Je ne sais pas si le `result.response` est rempli quoiqu'il arrive, mais je pense que ça peut être bien de se protéger ici

Reply:
@eliasdemnati I already check it in the if above no?

```
    if (ctx.params.stmpSelected === 'smtp1' && smtp1.awsSesMessageIdHost) {
```

Reply:
@eliasdemnati I already checked that all values are available before generating messageId no ?

>     if (ctx.params.stmpSelected === 'smtp1' && smtp1.awsSesMessageIdHost) {